aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -47,6 +60,24 @@ public static String testBucketName() {
     return System.getenv("AWS_TEST_BUCKET");
   }
 
+  /**
+   * Get the environment variable AWS_TEST_ACCESS_POINT for a default bucket to use for testing
+   * @return bucket name
+   */
+  public static String testAccessPointName() {
+    return System.getenv("AWS_TEST_ACCESS_POINT");
+  }
+
+  /**
+   * Get AccessPointARN for a default bucket to use for testing
+   * @return access point arn
+   */
+  public static String testAccessPointARN() {
```

Comment:
if we remove `testAccessPointName`, it means this can also be removed, or moved into `TestS3FileIOIntegration`